src/container.cpp

```diff
@@ -724,6 +724,7 @@ void Container::internalAddThing(uint32_t, Thing* thing)
 
 void Container::startDecaying()
 {
+	g_game.startDecay(this);
```

Comment:
Item::startDecaying()?
This is override method, so making use of the base class method would make more sense imo.

Reply:
Yes. Much cleaner. Fixed.